src/templates/static/get_started/binary_options_mt5.jsx

```diff
@@ -115,6 +113,42 @@ const BinaryOptionsForMT5 = () => (
 
                     <SeparatorLine invisible show_mobile className='gr-padding-10' />
                 </Section>
+                <Section id='type-of-trades' header={it.L('Type of trades')}>
```

Comment:
I think it should be renamed to `Types of trades` to match what we have in get-started / binary options